client/src/app/site/pages/organization/pages/accounts/pages/account-list/components/account-list/account-list.component.ts

```diff
@@ -139,6 +142,43 @@ export class AccountListComponent extends BaseListViewComponent<ViewUser> {
         }
     }
 
+    public async assignHomeCommitteeToUsers(): Promise<void> {
+        const title = this.translate.instant(`This will add or remove the selected accounts to the selected home committee:`);
+        const ADD = _(`Add`);
+        const REMOVE = _(`Remove`);
+        const actions = [ADD, REMOVE];
+        const committees = this.committeeRepo.getViewModelList();
+        const result = await this.choiceService.open<ViewCommittee>({
+            title,
+            choices: committees,
+            actions,
+            content: this.translate.instant(
+                `Attention: Accounts will overwrite other home committees if possible and remove the external status.`
+            )
+        });
+        if (result) {
+            if (!result.firstId) {
```

Comment:
Why throw an error when this is not needed for `REMOVE`? 

Additionally, I don't see that this gets handled anywhere. Can this even happen?

Reply:
Hmm, yes, I removed the result.firstId error check. This should not happen.